recipes/yt/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py --quiet install --single-version-externally-managed --record=record.txt
```

Comment:
So, this is our standard. Ideally, we will want Windows to do this too, but there is a `conda build` bug that blocks that for now.